Python.gitignore

```diff
@@ -133,3 +133,11 @@ dmypy.json
 
 # pytype static type analyzer
 .pytype/
+
+# IDE
+.vscode
+.idea
+
+# static
```

Comment:
> When `python manage.py collectstatic` is invoked, a static directory is created. So that's that.

Given this is the first mention of adding these directories, it'd be great to hear that others are also encountering this and would benefit from the change. Are you sure there's no other resources or discussions about this out there?

Also, are we able to better document what this group of rules is about?

Reply:
Can't say for sure about the resources or discussions actually.

But about the documentation, I guess what we can say about them is this:

    static files generated from Django application using `collectstatic`

Reply:
The other concern I have is that these names are rather broad (in particular `media`), and might inadvertently catch a user-created directory with the same name. Can these be opt-in (and commented out) or do you not feel this is a concern?

Reply:
Commenting them out might just arise the need for a developer to rewrite them at the end of the file as it might be the case that one might not read the whole `.gitignore` file; I know I don't.

And anyway, the name `media` is not usually used for a source code related stuff and is probably general enough for one to realize to not use.